packages/@lwc/template-compiler/src/parser/attribute.ts

```diff
@@ -83,21 +84,27 @@ export function isFragmentOnlyUrl(url: string): boolean {
 export function normalizeAttributeValue(
     attr: parse5.Attribute,
     raw: string,
-    tag: string
+    tag: string,
+    ctx: ParserCtx,
```

Comment:
I would recommend passing the context as the first argument. This will match the function signature in `index.ts`.